pkg/scaffold/v2/dockerfile.go

```diff
@@ -57,8 +57,10 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager
 
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:latest
+FROM gcr.io/distroless/static:nonroot 
 WORKDIR /
 COPY --from=builder /workspace/manager .
-ENTRYPOINT ["/manager"]
-`
+EXPOSE 443
+EXPOSE 8443
```

Comment:
?

Reply:
Trying to check if by exposing the port we could solve this issue.

Reply:
To expose the port and solve the issue.

Reply:
Just testing